example-get-started-experiments/code/README.md

```diff
@@ -11,7 +11,7 @@ This is an auto-generated repository for use in [DVC](https://dvc.org)
 This is a Computer Vision (CV) project that solves the problem of segmenting out 
 swimming pools from satellite images. 
 
-![Example result](./results/evaluate/plots/images/REGION_1-24_0_1024_0_1024.png)
```

Comment:
This was pointing to a path that doesn't exist

Reply:
This is going to be an issue until we fix #188 , the paths are from a random rample